app/views/mailers/digest_mailer/digest_email.html.erb

```diff
@@ -15,7 +15,7 @@
 <div>
   👋 <b>Looking for work?</b> <a href="<%= app_url("/listings/jobs") %>">Browse job openings</a> or create an <a href="<%= app_url("/listings/forhire") %>">available for hire</a> listing.
   <br />
-  🔍 <b>Hiring developers?</b> Post your position(s) on <a href="<%= app_url("/page/post-a-job") %>"><%= community_name %> Listings</a>.
+  🔍 <b>Hiring  <%= community_members_label %>?</b> Post your position(s) on <a href="<%= app_url("/page/post-a-job") %>"><%= community_name %> Listings</a>.
```

Comment:
I‘m reviewing this on mobile so it’s hard to tell but it looks like there’s an extra space before the erb here.

Reply:
Removed the extra space thanks!

Reply:
yep, two spaces